frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/radiogram-card/radiogram-card-content-personnel-count/radiogram-card-content-personnel-count.component.ts

```diff
@@ -0,0 +1,36 @@
+import type { OnInit } from '@angular/core';
+import { Component, Input } from '@angular/core';
+import { Store } from '@ngrx/store';
+import type { PersonnelCountRadiogram } from 'digital-fuesim-manv-shared';
+import { UUID } from 'digital-fuesim-manv-shared';
+import type { Observable } from 'rxjs';
+import type { AppState } from 'src/app/state/app.state';
+import { createSelectRadiogram } from 'src/app/state/application/selectors/exercise.selectors';
+
+@Component({
+    selector: 'app-radiogram-card-content-personnel-count',
+    templateUrl: './radiogram-card-content-personnel-count.component.html',
+    styleUrls: ['./radiogram-card-content-personnel-count.component.scss'],
+})
+export class RadiogramCardContentPersonnelCountComponent implements OnInit {
+    @Input() radiogramId!: UUID;
+
+    radiogram$!: Observable<PersonnelCountRadiogram>;
+
+    readonly personnelCategories = [
+        'notarzt',
+        'notSan',
+        'rettSan',
+        'san',
+        'gf',
+    ] as const;
+
+    constructor(private readonly store: Store<AppState>) {}
+
+    ngOnInit(): void {
+        const selectRadiogram = createSelectRadiogram(this.radiogramId);
+        this.radiogram$ = this.store.select(
+            selectRadiogram
+        ) as Observable<PersonnelCountRadiogram>;
```

Comment:
I don't think you need to use a separate variable for `selectRadiogram`, and we have never used these selectors like this.